app/preview.py

```diff
@@ -63,7 +64,19 @@ def get_page_count(pdf_data):
 @auth.login_required
 def page_count():
     json = get_and_validate_json_from_request(request, preview_schema)
-    return jsonify({"count": get_page_count(get_pdf(get_html(json)).read())})
+    if json["template"]["letter_attachment"]:
+        attachment_page_count = json["template"]["letter_attachment"]["page_count"]
+    else:
+        attachment_page_count = 0
+    template_page_count = get_page_count(get_pdf(get_html(json)).read())
+    total_page_count = template_page_count + attachment_page_count
+    return jsonify({"count": total_page_count, "attachment_page_count": attachment_page_count})
```

Comment:
Yeah it gets stored in the DB: https://github.com/alphagov/notifications-api/blob/main/app/models.py#L2436

Reply:
Yeah, we have access to it in admin, but also I'm not against returning it from here (I was at the start, because it felt redundant). We are passing it in anyway with template object, and returning it back means we can keep all calculations in one place, and then get back a single dictionary with all page counts. It's kinda neat.